sens-test-pybamm.py

```diff
@@ -0,0 +1,63 @@
+import pybamm
```

Comment:
can this file be deleted or moved somewhere else?

Reply:
I didn't notice this one, I'll delete it